src/Shared/AIExtensions/DataContentMediaTypeExtensions.cs

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Microsoft.Extensions.AI;
+
+/// <summary>
+/// Provides extension methods for interpreting the media type of a <see cref="DataContent"/>.
+/// </summary>
+#if !SHARED_PROJECT
+[System.Diagnostics.CodeAnalysis.ExcludeFromCodeCoverage]
+#endif
+internal static class DataContentMediaTypeExtensions
+{
+    /// <summary>
+    /// Determines whether the <see cref="DataContent"/> represents an image.
+    /// </summary>
+    /// <param name="content">The <see cref="DataContent"/>.</param>
+    /// <returns><see langword="true"/> if the content represents an image; otherwise, <see langword="false"/>.</returns>
+    public static bool HasImageMediaType(this DataContent content)
+        => content.MediaType is { } mediaType && mediaType.StartsWith("image/", StringComparison.OrdinalIgnoreCase);
```

Comment:
```suggestion
        => content.MediaType?.StartsWith("image/", StringComparison.OrdinalIgnoreCase) is true;
```
?

If this is something we expect every consumer to need to do, should we bake it in with a helper instance method on DataContent, e.g.
```C#
public bool HasMediaTypePrefix(string prefix)
```
? ("Prefix" isn't ideal naming, but HasMediaTypeType doesn't sound right, either :))

Reply:
Yeah, adding a new instance member sounds fine to me 🙂